plugins/woocommerce/includes/abstracts/abstract-wc-product.php

```diff
@@ -1932,6 +1932,21 @@ public function single_add_to_cart_text() {
 		return apply_filters( 'woocommerce_product_single_add_to_cart_text', __( 'Add to cart', 'woocommerce' ), $this );
 	}
 
+	/**
+	 * Get the aria-describedby description for the add to cart button.
+	 *
+	 * @return string
+	 */
+	public function aria_describedby_description() {
```

Comment:
For consistency with the `add_to_cart_text()` and `add_to_cart_description()` functions, I think we should name this `add_to_cart_aria_describedby()` or similar, as it only applies in that context ("add to cart").